middleware/index.js

```diff
@@ -0,0 +1,55 @@
+const Campground = require("../models/campground");
```

Comment:
Always use absolute path. This is to prevent unexpected routing of files to the codebase that may result to some errors.